src/screens/signup/otpVerification/style.ts

```diff
@@ -1,6 +1,15 @@
+import { Dimensions } from 'react-native';
 import styled from 'styled-components/native';
+import applyScale from '../../../utils/applyScale';
+import { theme } from '../../../theme/types';
 
-export const Container = styled.View``;
+const { width } = Dimensions.get('window');
+
+export const Container = styled.View`
+  flex: 1;
+  width: ${width}px;
```

Comment:
No need to use phone width, flex 1, automatically makes the container full view on the page, taking all width and height of the screen, if for reason the width doesn't work then give the width a 100%